### ISSUE TITLE
BRs: Consider including EVG-style section anchors in the BR Effective Dates table

### ISSUE THREAD

**Issue by castillar** [cabforum/servercert#299]
The EVG Effective Dates table includes a hyperlink anchor for each effective date, linking the date statement to the exact section that mandates it. This is a really handy feature, and we should consider adding it to the BRs if it isn't too much hassle to manage.

**Comment by sleevi** [comment]
Yeah, I did this during the new Pandoc work mostly because the table was smaller, and it was an excuse to test out the link verifier.

The reason it was messier for the BRs is that we regularly have BR ballots that affect multiple sections, so I wasn’t sure how we wanted to handle those.

**Comment by dzacharo** [comment]
Addressed in SC095
